tests/core/lethe_grid_tools_point_line_distance.output

```diff
@@ -0,0 +1,37 @@
+
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
+DEAL::The distance is 1.00000
```

Comment:
It's a bit funny that the distance is 1 for your entire test. Is that what you expected? seems a bit like a simple test no?

Reply:
It is what I expect, because the test is placing point on a cylinder of radius 1 using the equation of a cylinder. Then it checks if the `find_point_line_distance` returns the right distance between the cylinder axis and the point, which always the radius of 1.

I can make it more complex by varying the radius, and maybe randomize the point positions?